modules/virt-using-windows-dvd-disk-image.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-using-windows-dvd-disk-image_{context}"]
+= Using a Windows DVD to create a VM disk image
+
+Microsoft does not provide disk images for download, but you can create a disk image using the Windows DVD. This disk image can then be used to create virtual machines.
+
+.Procedure
+
+. In the {VirtProductName} web console, click *Storage* -> *PersistentVolumeClaims* -> *Create PersistentVolumeClaim With Data upload form*.
```

Comment:
ditto

Reply:
Please review the resolved conversations and the preview link of the content. This is AsciiDoc formatting.